internal/controller/nonadminbackupstoragelocation_controller.go

```diff
@@ -760,10 +764,10 @@ func (r *NonAdminBackupStorageLocationReconciler) createVeleroBSL(ctx context.Co
 	// 1. Check if user has specified the prefix as "foo" in NaBSL creation, then prefix used would be <non-admin-ns>/foo
 	// 2. TODO use the value from enforced spec if specified by the admin, if specified as "bar" then prefix used would be <non-admin-ns>/bar
```

Comment:
NIT: remove TODO and update comment here

Reply:
done: https://github.com/migtools/oadp-non-admin/pull/236/commits/d0a0c666f2f5d8d4c6cc4a739c0c314daf977bc5